### DISCUSSION TITLE
Revoke Consent Session by Port-Forwarding

### DISCUSSION THREAD

**Discussion by pommelinho** [ory/hydra#2371] [Q&A] [answered]
Hey!

Hydra is running in a pod in our Kubernetes Cluster.

I want to revoke consent session for a subject by calling https://www.ory.sh/hydra/docs/reference/api#revokes-consent-sessions-of-a-subject-for-a-specific-oauth-20-client. 

I used port forwading by doing
`kubectl port-forward hydra-744dc9cff5-f7nv7  4445`

Then i do
`curl http://localhost:4445/oauth2/auth/sessions/consent?subject=4&all=true   -H "Content-Type: application/json" -H 'X-Forwarded-Proto: https' -k -X DELETE`

However i get the following error in my hydra logs

```
time=2021-02-26T17:14:06Z level=error msg=An error occurred while handling a request audience=application error=map[message:can not serve request over insecure http trace:
github.com/ory/hydra/x.RejectInsecureRequests.func1
	/home/ory/x/tls_termination.go:82
github.com/urfave/negroni.HandlerFunc.ServeHTTP
	/go/pkg/mod/github.com/urfave/negroni@v1.0.0/negroni.go:29
github.com/urfave/negroni.middleware.ServeHTTP
	/go/pkg/mod/github.com/urfave/negroni@v1.0.0/negroni.go:38
github.com/ory/x/metricsx.(*Service).ServeHTTP
	/go/pkg/mod/github.com/ory/x@v0.0.170/metricsx/middleware.go:251
github.com/urfave/negroni.middleware.ServeHTTP
	/go/pkg/mod/github.com/urfave/negroni@v1.0.0/negroni.go:38
github.com/ory/hydra/metrics/prometheus.(*MetricsManager).ServeHTTP
	/home/ory/metrics/prometheus/middleware.go:26
github.com/urfave/negroni.middleware.ServeHTTP
	/go/pkg/mod/github.com/urfave/negroni@v1.0.0/negroni.go:38
github.com/ory/x/reqlog.(*Middleware).ServeHTTP
	/go/pkg/mod/github.com/ory/x@v0.0.170/reqlog/middleware.go:134
github.com/urfave/negroni.middleware.ServeHTTP
	/go/pkg/mod/github.com/urfave/negroni@v1.0.0/negroni.go:38
github.com/urfave/negroni.(*Negroni).ServeHTTP
	/go/pkg/mod/github.com/urfave/negroni@v1.0.0/negroni.go:96
net/http.serverHandler.ServeHTTP
	/usr/local/go/src/net/http/server.go:2843
net/http.(*conn).serve
	/usr/local/go/src/net/http/server.go:1925
runtime.goexit

```

What am i doing wrong here?

**Comment by pommelinho** [comment, marked as answer]
Ok i solved the issue now. you can close the question. sorry for the inconvenience.

**Reply by vinckr** [reply]
Heh, sorry for the late (or not existing) response :sweat_smile: . 

Can I ask what the issue was, or how you resolved it?

**Reply by pommelinho** [reply]
Hey!
`curl -X DELETE   -H 'Accept: application/json' -H 'X-Forwarded-Proto: https' -k  "http://localhost:4445/oauth2/auth/sessions/consent?subject=4&client=gastromatic-terminal"` did the job
